scrnaseq.Rmd

```diff
@@ -17,9 +17,14 @@ params:
   # Data type ("RNA", "Spatial")
   assay_raw: "RNA"
   
-  # Downsample data to at most n cells per sample (mainly for tests)
+  # Downsample data to at most n cells per sample AFTER filtering (mainly for tests)
   #   NULL to deactivate
-  downsample_cells_n: 100
+  downsample_cells_n: NULL
+  
+  # Downsample all samples equally according to the smalles sample
+  #   TRUE/FALSE
+  #   Overwritten by downsample_cells_n
+  downsample_cells_equally: TRUE
```

Comment:
Should be set to FALSE.

Reply:
Not yet. We had `downsample_cells_n` set to 100 for this test report as long as we don't have a final release. It's faster to compile for testing :-). So I just switched to this and set `downsample_cells_n=NULL`.

Reply:
Okay so you rely on the test dataset having 100 cells ?

Reply:
okay